com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/CobolWorkspaceServiceImpl.java

```diff
@@ -50,16 +50,15 @@
  */
 @Slf4j
 @Singleton
-public class CobolWorkspaceServiceImpl implements CobolWorkspaceService {
-
+public class CobolWorkspaceServiceImpl implements WorkspaceService {
   private DataBusBroker dataBus;
   private Provider<LanguageClient> clientProvider;
 
   @Inject
   public CobolWorkspaceServiceImpl(DataBusBroker dataBus, Provider<LanguageClient> clientProvider) {
     this.dataBus = dataBus;
     this.clientProvider = clientProvider;
-    dataBus.subscribe(DataEventType.REQUIRED_COPYBOOK_EVENT, this);
+    dataBus.subscribe(DataEventType.REQUIRED_COPYBOOK_EVENT);
```

Comment:
We do not need this subscription at all due to CobolWorkspaceServiceImpl does not accept any events